src/jest-preset.ts

```diff
@@ -19,7 +19,6 @@ export default {
     fastly,
   },
   moduleNameMapper: {
-    "^fastly:.*":
-      "node_modules/jest-preset-fastly-js-compute/fastly-js-compute-mock.js",
+    "^fastly:.*": join(__dirname, "./fastly-js-compute-mock.js"),
```

Comment:
This change allows us to test the preset locally

Reply:
Looks a good idea!